int8 type
cwru_ariac/Part sourcePart
cwru_ariac/Part targetPart
float64[] joints
float64 timeout
float64 maxPlanningTime

# type list
int8 none					= 0
int8 planToHome				= 10
int8 planToPart				= 12
int8 pickPart				= 13
int8 placePart				= 14
int8 planToMovePart			= 15
int8 sendJointsValue		= 16
int8 getJointsState			= 17
int8 grab					= 18
int8 release				= 19
int8 isGripperAttached		= 21
int8 waitForGripperAttach	= 22
int8 setMaxPlanningTime		= 23
int8 getCurrentBasePose		= 24
int8 getCurrentGripperPose	= 25
int8 executeLastPlan		= 26
int8 moveClosestParts		= 27
---
bool success
int8 ERROR_CODE
bool gripperAttached
float64 maxPlanningTime
geometry_msgs/PoseStamped basePose
geometry_msgs/PoseStamped gripperPose
float64 planningTime
float64 executingTime
float64[] jointsState

# ERROR CODE
int8 NO_ERROR			= 0
int8 UNKNOWN_ERROR		= 100
int8 INVALID_TYPE		= 101
int8 WRONG_PARAMETER	= 102
int8 PLANNING_TIMEOUT	= 103
int8 EXECUTING_TIMEOUT	= 104
int8 UNREACHABLE		= 105
int8 GRIPPER_FAULT		= 106
int8 COLLISION			= 107
int8 PART_DROPPED		= 108